### PR TITLE
2 button component

### LINKED ISSUE #2: Button component
Create button component.

Expectation:
* Looks like in figma https://www.figma.com/file/RiGqOf7INiJEwHiuum8q0x/Landing?node-id=430%3A2
* Could accept all button's props.

### PR DESCRIPTION
**Pull request by kostyamospan**

### CODE REVIEW

File: libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    alignItems: 'center',
+    justifyContent: 'center',
+    boxShadow: 'inset 0px 0px 10px 2px rgba(0,0,0,0.6)',
+    boxSizing: 'border-box',
+  };
+});
+
+const ButtonText = styled(Text)<TextProps>(({ theme }) => {
+  return {
+    color: theme.colors.primary.white,
```
Comment (gymnasy55) [inline libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx]: You don't need to create a new component, you can use just `<Text color="primary.white">text</Text>`

File: libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  };
+});
+
+export const HollowButton = memo<HollowButtonProps>(
+  ({ title, capital = true, background, ...props }) => {
+    return (
+      <Box padding={'5px'}>
+        <GradientOuterBox
+          width={props.width ?? '266px'}
```
Comment (gymnasy55) [inline libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx]: You can set this value in destructuring

File: libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+          height={props.height ?? '96px'}
```
Comment (gymnasy55) [inline libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx]: You can set this value in destructuring

File: libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+const ButtonText = styled(Text)<TextProps>(({ theme }) => {
+  return {
+    color: theme.colors.primary.white,
+  };
+});
+
+export const HollowButton = memo<HollowButtonProps>(
+  ({ title, capital = true, background, ...props }) => {
+    return (
+      <Box padding={'5px'}>
```
Comment (gymnasy55) [inline libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx]: Please use tag button as button, not Box

File: libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+          width={props.width ?? '266px'}
+          height={props.height ?? '96px'}
+          display={'flex'}
+          alignItems={'center'}
+          justifyContent={'center'}
+          {...props}
+        >
+          <HollowBox>
+            <ButtonText size="xl" fontWeight={'700'}>
+              {capital ? title.toUpperCase() : title}
```
Comment (gymnasy55) [inline libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx]: Do not use title prop, use children and in css use condition in `text-transform`

**Review by gymnasy55 — CHANGES_REQUESTED** [review]

File: libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx (hunk 2 above)
Comment (kostyamospan) [reply libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx]: kak, tell me please

File: libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+          {...props}
+        >
+          <HollowBox>
+            <Text
+              size="xl"
+              fontWeight={'700'}
+              color="primary.white"
+              textTransform={capital ? 'capitalize' : undefined}
+            >
+              {title}
```
Comment (gymnasy55) [inline libs/landing-ui/src/lib/components/HollowButton/HollowButton.tsx]: no please remove title as a prop at all, use children instead

**Review by gymnasy55 — CHANGES_REQUESTED** [review]

**Review by gymnasy55 — APPROVED** [review]

**Pull request merged**